packages/blockchain-wallet-v4-frontend/src/scenes/Dex/Swap/Swap.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect } from 'react'
```

Comment:
No changes in terms of logic and layout. Just Swap part in extracted from Dex main file.